python/cudf/cudf/tests/test_stats.py

```diff
@@ -170,15 +170,14 @@ def test_exact_quantiles_int(int_method):
 def test_approx_quantiles():
     arr = np.asarray([6.8, 0.15, 3.4, 4.17, 2.13, 1.11, -1.01, 0.8, 5.7])
     quant_values = [0.0, 0.25, 0.33, 0.5, 1.0]
-    approx_results = [-1.01, 0.8, 0.8, 2.13, 6.8]
 
     gdf_series = Series(arr)
+    pdf_series = pd.Series(arr)
 
     q1 = gdf_series.quantile(quant_values, exact=False)
+    q2 = pdf_series.quantile(quant_values)
 
-    np.testing.assert_allclose(
-        q1.to_pandas().values, approx_results, rtol=1e-10
-    )
+    np.testing.assert_allclose(q1.to_pandas().values, q2.values, rtol=1e-10)
```

Comment:
Are we able to just do `assert_eq(q1, q2)` here? That tests that we have the right metadata for the Series as well.

Reply:
Yes.

Reply:
Sorry -- to be clear, I meant the `cudf.tests.utils.assert_eq` utility as opposed to numpy's `assert_equal`. Just `assert_eq` won't check for things like `lhs.index == rhs.index` and `lhs.name == rhs.name` for example.

Reply:
I've just been thinking about this PR in terms of keeping as much as possible the same. Lemme check.